include/trace/events/sof_intel.h

```diff
@@ -0,0 +1,48 @@
+#undef TRACE_SYSTEM
```

Comment:
@nklayman you need a commit title with the domain prefix, and a couple of lines for each commit message

suggested edit:

ASoC: SOF: Intel: add HDA interrupt source tracing

The Intel HDaudio controller relies on a single interrupt line which wire-ORs multiple interrupt sources, such as stream, IPC, SoundWire and wakes. This patch adds the ability to trace each event occurrence.

Reply:
The first commit has the domain prefix, and the rest will be squashed under that. Does it still matter if the commits have the prefix?

Reply:
@nklayman you have to split the code in 'logical' partitions. HDaudio interrupts can be one patch, the period_elapsed interrupt another. Each of the commits have to have the proper prefix for maintainers, this is typically used to identify who reviews the code.

Reply:
Should I split this into multiple PRs then? I was planning on squashing everything in this PR together but I now see that won't work.

Reply:
just keep one patch per logical function in the same PR @nklayman.

Reply:
These are two, mutually exclusive ways to use Github: 1) the "native Github way" versus 2) the "native git way". We already compared and discussed them in https://github.com/thesofproject/sof-test/pull/877 and in the links from there.

"Mutually exclusive" in the same PR. It is technically possible to use the Github way in one PR and the git way in another PR but that confuses reviewers greatly; as just demonstrated here. So most projects standardize on a single way. thesoftproject doesn't use the Github way right now and neither does https://github.com/zephyrproject-rtos/zephyr/issues/39194